gapic-generator-java/DEVELOPMENT.md

```diff
@@ -155,6 +155,32 @@ To generate a production GAPIC API:
    ```
    and then rebuild gapic-generator-java (`mvn clean install`).
 
+## Debugging the Plugin under googleapis with local gapic-generator-java (end-to-end debugging)
+
+To debug the local generator starting from protobuf compiler plugin
+
+1. Add [Remote JVM debug configuration](https://www.jetbrains.com/help/idea/tutorial-remote-debug.html) and breakpoints for the local generator in Intellij.
+
+2. In [googleapis](https://github.com/googleapis/googleapis) root directory, run the following commands:
+   1. Set `JVM_DEBUG_PORT` environment variable.
+       ```shell
+       export JVM_DEBUG_PORT=5005
+       ```
+   2. Run the command to build a client library, e.g., `java-monitoring`, with `--subcommands` flag to output commands spawned by `bazel build`.
+      ```shell
+      bazel build --subcommands //google/monitoring/v3:google-cloud-monitoring-v3-java
+      ```
+   3. In the output, try to find the command like the below one, the `darwin_arm64-opt-exec-2B5CBBC6` part maybe different on your environment.
```

Comment:
```suggestion
   3. In the output, find the command of protoc invocation with gapic-generator-java plugin. (the `darwin_arm64-opt-exec-2B5CBBC6` part maybe different on your environment.)
```

Remove unnecesary "try to".

Explain what is that command rather than "like the below".

"the `darwin_arm64-opt-exec-2B5CBBC6` ..." is a run-on sentence.

Reply:
done